cs.py

```diff
@@ -203,9 +202,15 @@ def main():
     command = sys.argv[1]
     kwargs = defaultdict(set)
     flags = set()
+    args = dict()
     for option in sys.argv[2:]:
         if option.startswith('--'):
-            flags.add(option.strip('-'))
+            option = option.strip('-')
+            if '=' in option:
+                key, value = option.split('=', 1)
+                args = {key: value}
```

Comment:
did you mean `args[key] = value`? We're only interested in `region` right now but it'd be nice to prevent it from being overwritten by another unsupported param (e.g. `--region=foo --bar=baz`)